### ISSUE TITLE
Children rendered multiple times

### ISSUE THREAD

**Issue by daaku** [darenju/react-flip-page#85]
I'm not really sure why, but it seems like the children of `FlipPage` are rendered several times. Here's a sample that logs when each child is rendered:

https://codesandbox.io/s/r778z39z5o

**Comment by darenju** [comment]
Hi,

This is strange, but not that surprising because the pages have to be rendered several times to be flipped. But, I see you are not using the `FlipPageItem` component. You should try to use it, it might help.

**Comment by slevy85** [comment]
Hi,
I think it is related to #42.

**Comment by namannehra** [comment]
@darenju I have made a react component similar to this one. I ran into the same issue. It's not possible to create the page turning animation without rendering the children multiple times but some issues can be resolved by hiding duplicates properly. Right now it's possible to move focus to a duplicate hidden child by hitting the tab key. This will cause issues for keyboard navigation and screen readers. This can resolved by setting `visibility: hidden` in CSS for duplicate children when their is no page turn.
